python/lsst/pipe/tasks/calibrateImage.py

```diff
@@ -424,12 +426,33 @@ def runQuantum(self, butlerQC, inputRefs, outputRefs):
             config=self.config.photometry_ref_loader, log=self.log)
         self.photometry.match.setRefObjLoader(photometry_loader)
 
-        outputs = self.run(id_generator=id_generator, **inputs)
-
-        butlerQC.put(outputs, outputRefs)
+        # This should not happen with a properly configured execution context.
+        assert not inputs, "runQuantum got more inputs than expected"
```

Comment:
If/when this falls over, will the error message specify that the `runQuantum` being talked about is the one from `calibrateImage`?

Reply:
Yes, the assert will give where it happened. This is a "this should absolutely never happen" error, hence the bare assert.